### DISCUSSION TITLE
Histogram bug

### DISCUSSION THREAD

**Discussion by NikitaSakharov** [mtex-toolbox/mtex#987] [Ask Anything] [answered]
It seems to me there is some tiny mistake in histogram function.

the first picture is output of 
histogram(mergedGrains,mergedGrains.equivalentRadius)

and the second one is from 
hist(mergedGrains,mergedGrains.equivalentRadius)

So in the 1st one we should fix it by multiply y-axis by 100
![1](https://user-images.githubusercontent.com/62562246/116054792-bbb49500-a684-11eb-9690-7d1db6d908fd.jpg)
![2](https://user-images.githubusercontent.com/62562246/116054796-bc4d2b80-a684-11eb-909b-92841ce5f856.jpg)

.

**Comment by ralfHielscher** [comment, marked as answer]
Unfortunately, Matlab histogram function does not allow to scale the y-axis in percent. 
On the other hand we can simply change the labels by
```matlab
yticklabels(mtexFig.gca,yticks*100)
```
Ralf.

**Reply by NikitaSakharov** [reply]
Thanks!

My workaround was in changing histogram function code (I've just changed the title to 'relative area fraction')
